packages/rum-recorder/src/domain/rrweb/mutation.ts

```diff
@@ -281,7 +281,7 @@ export class MutationBuffer {
         }
       }
       if (!node) {
-        for (let index = addList.length - 1; index >= 0; index += 1) {
+        for (let index = addList.length - 1; index >= 0; index -= 1) {
```

Comment:
could it be a good opportunity to add a test?

Reply:
I'll see what I can do